packages/flutter/test/widgets/binding_first_frame_rasterized_test.dart

```diff
@@ -0,0 +1,39 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:quiver/testing/async.dart';
+
+void main() {
+  setUp(() {
+    WidgetsFlutterBinding.ensureInitialized();
+  });
+
+  test('Deferred frames will trigger the first frame callback', () {
+    FakeAsync().run((FakeAsync fakeAsync) {
+      final WidgetsBinding binding = WidgetsBinding.instance;
```

Comment:
Remove the setup and just say

```suggestion
      final WidgetsBinding binding = WidgetsFlutterBinding.ensureInitialized();
```